lib/identity/helpers/auth.rb

```diff
@@ -50,8 +51,8 @@ def authorize(params, confirm=false)
         if !authorization
           authorization = authorizations.detect { |a|
             a["client"] && a["client"]["legacy_id"] &&
-              a["client"]["legacy_id"] == params["client_id"] &&
-              a["scope"] == (params["scope"] || ["global"])
+              a["client"]["legacy_id"] == client_id &&
+              a["scope"]               == (params["scope"] || ["global"])
```

Comment:
Align the operands of an expression in an assignment spanning multiple lines.<br>Operator `==` should be surrounded with a single space.